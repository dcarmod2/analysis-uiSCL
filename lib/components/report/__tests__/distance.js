//
import enzyme from 'enzyme'
import React from 'react'

import Distance from '../distance'

describe('Report > Distance', () => {
  it('renders correctly', () => {
    const props = {
      km: 2
    }

    // mount component
    const tree = enzyme.shallow(<Distance {...props} />)
    expect(tree).toMatchSnapshot()
  })
})
