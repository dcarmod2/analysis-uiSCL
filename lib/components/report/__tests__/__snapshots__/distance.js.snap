// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Report > Distance renders correctly 1`] = `
<span>
  2 km
   (
  1.2 mi
  )
</span>
`;
